src/l500/CMakeLists.txt

```diff
@@ -21,6 +24,9 @@ target_sources(${LRS_TARGET}
         "${CMAKE_CURRENT_LIST_DIR}/l500-fw-update-device.h"
         "${CMAKE_CURRENT_LIST_DIR}/l500-serializable.h"
         "${CMAKE_CURRENT_LIST_DIR}/l500-options.h"
+        "${CMAKE_CURRENT_LIST_DIR}/l535-options.h"
```

Comment:
shouldn't this be `l535-device-options.h`?

Reply:
done